templates/csi-templates/csi-rbd-secret.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ csi_rbd_secret| default ('csi-rbd-secret') }}
+  namespace: {{ csi_rbd_secret_ns| default('default') }}
+data:
+  # Key value corresponds to a user name defined in Ceph cluster
+  # ceph auth get-key admin | base64
+  admin: {{ base64_encoded_admin_password }}
+  # Key value corresponds to a user name defined in Ceph cluster
+  # # ceph auth get-key kubernetes | base64
+  kubernetes: {{ base64_encoded_user_password }}
```

Comment:
are we going to comment this param ? may be for now. Later we can update this yaml with expected kubernetes value.

Reply:
I am ok with commenting this param,but we need approval from other reviewers.

Reply:
@RazTamir @ebenahar should we comment out the section for **kubernetes** user as ramkiperiy suggested. I don't think we all are creating a kubernetes user, currently. Maybe we can uncomment it once we start creating separate users. 

Please suggest @keesturam @prsurve @ramkiperiy

Reply:
I'm not yet familiar with this attribute so can't tell what is the right way but,
what is the expected behavior from our customers?
Each customer will need to create a K8s user?
What happens if not? is it ok to comment this?

Reply:
@RazTamir so by default it comes uncommented, so we are expected to create a user. That user name can be anything, but for now, the csi yamls have hard-coded it to **kubernetes**. Maybe we can confirm this from CSI developers and ceph developers.